server/modules/getRefreshTokenAndUpdatePlaylist.js

```diff
@@ -0,0 +1,31 @@
+var request = require("request");
+const { Pool } = require("pg");
+const deleteAllTracksFromSpotifyPlaylist = require("./deleteAllTracksFromSpotifyPlaylist");
+const addTracksToPlaylist = require("./addTracksToPlaylist");
+
+// /update route
+const getRefreshAndUpdatePlaylist = async function (req, res) {
+  // calling the /refresh_token endpoint to call the function to get the access and refresh tokens
+  const serverURL = process.env.server_url;
+  const requestRefreshToken = await fetch(`${serverURL}/refresh_token`);
+  const newTokenObject = await requestRefreshToken.json();
+  const newAccessToken = newTokenObject.access_token;
+  console.log(`NEWWWWWWWWW access token: ${newAccessToken}`);
+
+  // API call to delete tracks from yesterday's spotify playlist
+  const playlistID = process.env.playlist_id;
+  deleteAllTracksFromSpotifyPlaylist(newAccessToken, playlistID);
+
+  // API call to add tracks randomly selected from db to spotify playlist
+  const getRandomSongs = await fetch(`${serverURL}/songs/random`);
```

Comment:
@christina-mifsud @munozirianni1988 @ManSangSin 
Thought/Suggestion: On testing the whole function, I can successfully add 5 random songs to the playlist - woo it's awesome! I noticed that sometimes the same song gets added to the array, in future we might want to add a bit of code so a song doesn't appear again for the next day/week?
i.e. 
Day 1: Song 1, Song 2, Song 3
Day 2: Song 2, Song 4, Song 5
Day 3: Song 6, Song 3, Song 2